### ISSUE TITLE
Mapping between Test262 and web-features

### ISSUE THREAD

**Issue by foolip** [web-platform-dx/web-features#814]
I have just filed https://github.com/tc39/test262/issues/4043. Filing this issue here so that web-features watchers are aware of it.

**Comment by captainbrosset** [comment]
@jugglinmike said this has been done and the issue referenced above will get closed.

**Comment by jugglinmike** [comment]
That's right--I just now closed the Test262 issue with some references to where the work happened (we did the work in 2025 but simply forgot that issue existed). This issue is safe to close, too!
